ddtrace/profiling/exporter/pprof.pyx

```diff
@@ -373,11 +388,14 @@ class PprofExporter(exporter.Exporter):
     ):
         # type: (...) -> _stack_event_group_key_T
 
-        if event.trace_type == ext.SpanTypes.WEB.value:
-            trace_resource = self._none_to_str(event.trace_resource)
-        else:
-            # Do not export trace_resource for privacy concerns.
-            trace_resource = ""
+        trace_resource = None
+        # Do not export trace_resource for non Web spans for privacy concerns.
+        if (
+            event.trace_resource_container is not None
+            and len(event.trace_resource_container) > 0
```

Comment:
```suggestion
            and event.trace_resource_container
```

Python list truthiness is whether it is empty or not.

Reply:
and same for `None`, so you can just do:

```python
if event.trace_resource_container and event.type_type == ext.SpanTypes.WEB.value:
    trace_resource = ...
```

Reply:
thanks!